sources/packages/backend/apps/queue-consumers/src/processors/schedulers/esdc-integration/msfaa-integration/_tests_/msfaa-part-time-process-response-integration.scheduler.e2e-spec.ts

```diff
@@ -157,6 +166,23 @@ describe(
       // Validate second confirmed record.
       expect(secondSignedMSFAA.dateSigned).toBe("2021-11-22");
       expect(secondSignedMSFAA.serviceProviderReceivedDate).toBe("2021-11-23");
+
+      // Get the notification using the student and notification message id. Since a fake student is created for every test run, it will ensure uniqueness.
+      // Expecting one notification for the cancelled MSFAA record.
```

Comment:
https://github.com/bcgov/SIMS/pull/2069/files#r1248261900

Reply:
The record status is already being validated a few lines below:
 
`expect(cancelledMSFAA.cancelledDate).toBe("2021-11-24");`      
`expect(cancelledMSFAA.newIssuingProvince).toBe("ON");`